packages/block-editor/src/components/block-navigation/index.js

```diff
@@ -62,16 +62,15 @@ export default compose(
 		const {
 			getSelectedBlockClientId,
 			getBlockHierarchyRootClientId,
-			getBlock,
-			getBlocks,
+			__unstableGetBlockWithControlledInnerBlocks,
+			__unstableGetBlocksWithControlledInnerBlocks,
 		} = select( 'core/block-editor' );
 		const selectedBlockClientId = getSelectedBlockClientId();
 		return {
-			rootBlocks: getBlocks( '', { includeControlledInnerBlocks: true } ),
+			rootBlocks: __unstableGetBlocksWithControlledInnerBlocks( '' ),
```

Comment:
Would something like `getBlocksTree` be more clear / succinct? Not sure "with controlled inner blocks" clarifies much at a glance.

Reply:
Yeah, I think that would be a better approach. Indicating that this is "all blocks in the tree of the editor," rather than the blocks attached to a specific block parent. I was struggling over the naming here, so thanks for the suggestion!